compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1958,3 +1958,6 @@ error.worker.send.action.not.allowed.in.lock.statement=\
 
 error.worker.receive.action.not.allowed.in.lock.statement=\
   using receive action within the lock statement is not allowed to prevent possible deadlocks
+
+error.empty.regexp.string.disallowed=\
+  regular expression with empty string is not allowed
```

Comment:
```suggestion
  regular expression is not allowed: empty RegExp
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39848/commits/927c1a6378676f68f309748de84c9bb66f76f2f9.